### PR TITLE
12/impact section

### PR DESCRIPTION
**Pull request by NewtonOmbese**
# Description

Created a section that describes the impact section that shows an image on the right side, a paragraph of text and a title. This component displays provided text and images side by side on the social impact section using the reusable animated button components in issue https://github.com/italanta/elewa-group/issues/11. 

To achieve this, I needed to:
- Step 1: Create the home impact section component
```
nx g @nrwl/angular:component pages/home/home-impact-section
```
- Make an import of the button module to be reused in the component

Fixes # (issue  https://github.com/italanta/elewa-group/issues/12) 

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Desktop View
impact-section

Mobile View
Screenshot from 2023-02-22 11-41-22
Screenshot from 2023-02-22 11-41-36

Video 
https://user-images.githubusercontent.com/79839603/220568216-961f6a57-56c9-4746-9493-9114a60776d6.mp4

# How Has This Been Tested?
Step 1: Added the component to home-page-component.html
Step 2: Started the server by running ```nx serve elewa-group-website```
Step 3: Using chrome developer tools to test on the responsiveness of the website

# Checklist:t
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by NewtonOmbese** [comment]
Review and give feedback @IanOdhiambo9

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 1]
```diff
@@ -12,13 +12,17 @@ import { CardsModule } from '@elewa-group/features/components/cards';
 import { PrevDirective } from './components/Directives/prev.directive';
 import { NextDirective } from './components/Directives/next.directive';
 
+import { HomeImpactSectionComponent } from './components/home-impact-section/home-impact-section.component';
+import { ButtonsModule } from '@elewa-group/features/components/buttons';
+
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:17]: fix order of imports

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Use the right image and provide a screenshot of your latest changes on both mobile and web. I can't view the button on the mobile screenshot you provided. also sync the latest updates

**Comment by NewtonOmbese** [comment]
@LemmyMwaura  fixed the import issues

**Comment by NewtonOmbese** [comment]
@LemmyMwaura  the image link was not provided plus also for the second reusable is oversized due to the length of its text content

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
     <div mainPage class="home-page">
       <elewa-group-elewa-hero></elewa-group-elewa-hero>
       <elewa-group-home-hero-section></elewa-group-home-hero-section>
+      <elewa-group-home-impact-section></elewa-group-home-impact-section>
+      <elewa-group-home-jobs-section></elewa-group-home-jobs-section>
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:7]: One of the jobs sections should be removed

**Comment by LemmyMwaura** [comment]
> @LemmyMwaura the image link was not provided plus also for the second reusable is oversized due to the length of its text content

Okay the image will be updated later but the 'learn more' button should be full-width on mobile - fix that.

**Comment by NewtonOmbese** [comment]
@LemmyMwaura  I can resize them since they are reusable buttons

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html (hunk 2 above)
Comment (NewtonOmbese) [reply libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:7]: They are different component the jobs section being a componnt that was merged and mine being impact section

Comment (LemmyMwaura) [reply libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:7]: Check the file please. There are two job sections. One shouldn't be there, check the design for the right placement.

Comment (NewtonOmbese) [reply libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html]: Sure thing. I've fixed it

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
The reusable button needs a min-width of 100% so it can inherit size from the parent.

**Comment by NewtonOmbese** [comment]
@IanOdhiambo9  implemented the min width using both `ngStyle` and `SCSS` but didn't work

**Pull request closed without merging**